src/NuGetUtility/PackageInformationReader/PackageInformationReader.cs

```diff
@@ -1,4 +1,7 @@
-﻿using NuGet.Protocol.Core.Types;
+using NuGet.Configuration;
```

Comment:
Please properly abstract NuGet Protocol stuff. This allows for much easyer testing